### ISSUE TITLE
Configuration for 19041.450

### ISSUE THREAD

**Issue by minichate** [stascorp/rdpwrap#1142]

**Comment by naseblau** [comment]
allow RDP-File @ Windows-Defender

**Comment by trjwaugh** [comment]
![rdp](https://user-images.githubusercontent.com/38141930/90400737-e87eca00-e06a-11ea-93e1-b2d7834b1dc6.JPG)

Any help for this?

**Comment by hokhosh** [comment]
> ![rdp](https://user-images.githubusercontent.com/38141930/90400737-e87eca00-e06a-11ea-93e1-b2d7834b1dc6.JPG)
> 
> Any help for this?

Same problem here!

**Comment by affinityv** [comment]
No, the title is wrong.... you need suport for 10.0.19041.84, which is the termsrv.dll version.  This support is available in 3rd party INI, like my own one [here](https://raw.githubusercontent.com/affinityv/INI-RDPWRAP/master/rdpwrap.ini)  Once the INI file has been replaced, do `RDPWInst -r` to restart RDP, if that fails, perhaps a restart, if that still fails, then it /may/ be that Windows Defender or other AV is interfering.  Aside from those things, make sure that the INI file is in normal DOS format.

**Comment by maaraplus** [comment]
No, I have your ini file with the support for 10.0.19041.84, it is all green and it still does not work for my windows version 10.0.19041.450.

**Comment by samuk190** [comment]
> No, I have your ini file with the support for 10.0.19041.84, it is all green and it still does not work for my windows version 10.0.19041.450.

it worked here with -> Version	10.0.19041 compilation 19041

Which is claerly enough I dont need update my windows for a long time I just updated to use wsl2

**Comment by maaraplus** [comment]
Well, thank you, but as this was a migration from 1909 to 2004 (and I was checking that there is an INI file for 2004 available), then the only option for me now is to uninstall the whole 2004 update, which I would rather skip. Somewhere between 19041.84 and 19041.450 the binary offsets have changed again...

**Comment by samuk190** [comment]
> Well, thank you, but as this was a migration from 1909 to 2004 (and I was checking that there is an INI file for 2004 available), then the only option for me now is to uninstall the whole 2004 update, which I would rather skip. Somewhere between 19041.84 and 19041.450 the binary offsets have changed again...

I highly recommend upgrading only when there's something new which is very essential. For me hyper v + wsl2 is the essential as I can run virtualbox with my docker desktop in wsl2 <3

**Comment by affinityv** [comment]
2004 works, many people are using it.  Update as regularly as possible for regular SECURITY updates, delay feature updates (you need pro version to be able to do so).
If the INI file isn't working for you, then it is likely one of a few things, you have it saved without DOS line endings -- what do you see if you "type rdpwrap.ini" in a cmd window?  Does notepad open it up and it looks good in notepad?
Is Windows Defender active and if so, have you got exceptions for the "C:\Program Files\RDP Wrapper" directory?
Are any other AV or security software causing issues that need exceptions?

**Comment by maaraplus** [comment]
![image](https://user-images.githubusercontent.com/69896460/90730117-e1a0c300-e2c7-11ea-9c14-41d4728e830a.png)
The INI file seems to be fine. I have Avast as the antivirus but I am using RDPwrap with it for seven years, so it would be a real coincidence that it stopped working exactly now. I was going through regular updates of the INI file after major windows updates in the past and it kept working. Do you really think that there is no change between 10.0.19041.84 (which was a preview version of the 2004 update for insiders back in February) and 10.0.19041.450 which is the current iteration of it being dispatched now?

**Comment by maaraplus** [comment]
My foolish mistake. Only when I posted the screenshot to github, I realized that the wrapper got somehow uninstalled during the windows update. I reinstalled it and voila, it works! Kudos to your great work affinityv.

**Comment by samuk190** [comment]
> My foolish mistake. Only when I posted the screenshot to github, I realized that the wrapper got somehow uninstalled during the windows update. I reinstalled it and voila, it works! Kudos to your great work affinityv.

Nice! Just a tip...
Avast and Windows defender are the same, with some windows defender features being much better. The big difference here is that windows defender doesnt use much resources while avast use alot.. 
 >.< (I'm not using any third party antivirus since I got my pc) 
>.< (You can disable virus manually removing it's permissions (read,write,execute) and restarting pc.
>.<(You can search for viruses (running at memory) by using process manager selecting "VirusTotal Scan") 
--> Process explorer 64 manager is from microsoft and you can replace traditional task manager for it.

**Comment by PizzaProgram** [comment]
> This support is available in 3rd party INI, like my own one [here](https://raw.githubusercontent.com/affinityv/INI-RDPWRAP/master/rdpwrap.ini) Once the INI file has been replaced, do `RDPWInst -r` to restart RDP

@affinityv
Many Thanks for this INI file :) Works perfectly now!
